src/plugins/workspace/public/components/workspace_menu/workspace_menu.tsx

```diff
@@ -122,103 +135,111 @@ export const WorkspaceMenu = ({ coreStart, registeredUseCases$ }: Props) => {
                   position="right"
                   content={currentWorkspaceName}
                 >
-                  <EuiText size="s" style={{ maxWidth: '195px' }} className="eui-textTruncate">
+                  <EuiText
+                    textAlign="center"
+                    style={{ maxWidth: '200px' }}
+                    className="eui-textTruncate"
+                  >
                     {currentWorkspaceName}
                   </EuiText>
                 </EuiToolTip>
-              </EuiFlexItem>
-              <EuiFlexItem grow={false} data-test-subj="workspace-menu-current-use-case">
-                <EuiText size="s">{getUseCase(currentWorkspace)?.title ?? ''}</EuiText>
-              </EuiFlexItem>
-              <EuiFlexItem grow={false}>
-                <EuiSmallButton
-                  color="text"
-                  onClick={() => {
-                    closePopover();
-                    navigateToWorkspaceDetail(coreStart, currentWorkspace.id);
-                  }}
+                <EuiText
+                  size="xs"
+                  data-test-subj="workspace-menu-current-use-case"
+                  textAlign="center"
+                  color="subdued"
                 >
-                  {manageWorkspaceButton}
-                </EuiSmallButton>
+                  {getUseCase(currentWorkspace)?.title ?? ''}
+                </EuiText>
+              </EuiFlexItem>
+
+              <EuiFlexItem grow={false} style={{ width: '100%' }}>
+                <EuiFieldSearch
+                  value={querySearch}
+                  onChange={(e) => handleSearchInput({ text: e.target.value })}
+                  placeholder={searchFieldPlaceholder}
+                />
               </EuiFlexItem>
             </>
           ) : (
             <>
               <EuiFlexItem grow={false}>
-                <EuiAvatar size="m" color="plain" name="spacesApp" iconType="spacesApp" />
+                <EuiIcon size="xl" color="subdued" type="wsSelector" />
               </EuiFlexItem>
               <EuiFlexItem grow={false} data-test-subj="workspace-menu-current-workspace-name">
-                {currentWorkspaceName}
+                <EuiText textAlign="center">{currentWorkspaceName}</EuiText>
               </EuiFlexItem>
-              <EuiFlexItem grow={false}>
-                <EuiSmallButton
-                  color="text"
-                  onClick={() => {
-                    closePopover();
-                    coreStart.application.navigateToApp(WORKSPACE_LIST_APP_ID);
-                  }}
-                >
-                  {manageWorkspacesButton}
-                </EuiSmallButton>
+
+              <EuiFlexItem grow={false} style={{ width: '100%' }}>
+                <EuiFieldSearch
+                  value={querySearch}
+                  onChange={(e) => handleSearchInput({ text: e.target.value })}
+                  placeholder={searchFieldPlaceholder}
+                />
```

Comment:
Do we need to declare the render of `EuiFieldSearch` twice based on `currentWorksapce`? I think their properties are totally same.

Reply:
Thanks for the review, updated.